assets/javascripts/discourse/components/category-drop-assigned.js.es6

```diff
@@ -0,0 +1,18 @@
+import CategoryDrop from "select-kit/components/category-drop";
+
+export const NO_CATEGORIES_ID = "no-categories";
```

Comment:
Is this being used anywhere?

Reply:
we are using this in `else if` statement

Reply:
There's no need to export it then. Also, looks like it's already defined [in core](https://github.com/discourse/discourse/blob/c200238bdc241586bfe82b2d0fc1f600fec45b1f/app/assets/javascripts/select-kit/addon/components/category-drop.js#L10-L11) so you might want to import it instead ;)

Reply:
Right, ill add an import statement for this😃